playbooks/freeipa.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: server freeipa
```

Comment:
Why don't we need `become: true` here for setting up `repositories.yml` inside the `common` role?